supervisor/errata_utils.py

```diff
@@ -99,16 +99,36 @@ def get_erratum(erratum_id: str | int, full: bool = False) -> Erratum | FullErra
     )
 
     if full:
-        # fetching comments for the erratum
+        builds = get_erratum_builds(erratum_id)
         comments = get_erratum_comments(erratum_id)
         return FullErratum(
             **base_erratum.__dict__,
+            builds=builds,
             comments=comments,
         )
     else:
         return base_erratum
 
 
+def get_erratum_builds(erratum_id: str | int) -> list[str]:
+    """Get all builds (NVRs) attached to an erratum."""
+    logger.debug("Getting builds for erratum %s", erratum_id)
+    data = ET_api_get(f"erratum/{erratum_id}/builds")
+
+    builds_set = set()
+    for product_data in data.values():
+        if isinstance(product_data, dict) and "builds" in product_data:
```

Comment:
What I don't like this sort of defensive unpacking of the structure is that if we don't fully understand the schema, or there's a bug, we just silently get no builds. I'd rather just go for it, then we'll get a more informative exception - assume that it's a dict with "builds"